RoomLocator/RoomLocator.Api/Controllers/ModcamController.cs

```diff
@@ -0,0 +1,46 @@
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Runtime.Serialization.Json;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using RoomLocator.Data.Services;
+using RoomLocator.Domain.ViewModels;
+
+namespace RoomLocator.Api.Controllers
+{
+    [ApiVersion("1.0")]
+    [Route("api/v{version:apiVersion}/[controller]")]
+    [ApiController]
+    public class ModcamController :Controller
+    {
+        private readonly IHttpClientFactory _clientFactory;
+
+        public ModcamController(IHttpClientFactory clientFactory)
+        {
+            _clientFactory = clientFactory;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<ModcamInstallationsViewModel>>> GetPeopleCounterInstallations()
+        {
+            var url = "https://eds.modcam.io/v1/peoplecounter/installations";
+            var request = new HttpRequestMessage(HttpMethod.Get, url);
+            ModcamCredentialsService mc = new  ModcamCredentialsService();
```

Comment:
`mc` er ikke et specielt godt navn, det er ikke så specifikt for hvad det er, så kald den hellere noget der giver bedre mening, fx `modcamCredentialsService`

Derudover så er din første `ModcamCredentialsService` redundant. Jeg ville foretrække at du bruger `var` når typen er meget specific, som den er her. Fx

```cs
var modcamCredentialsService = new ModcamCredentialsService();
```